packages/p2p-v2/src/components/Checklist/Checklist.tsx

```diff
@@ -29,7 +30,12 @@ const Checklist = ({ items }: { items: TChecklistItem[] }) => {
                         <Button
                             className='p2p-v2-checklist__item-button'
                             disabled={item.isDisabled}
-                            icon={<ArrowRightIcon className='p2p-v2-checklist__item-button-icon' />}
+                            icon={
+                                <ArrowRightIcon
+                                    className='p2p-v2-checklist__item-button-icon'
+                                    {...(item.testId && { 'data-testid': item.testId })}
+                                />
+                            }
```

Comment:
need to discuss with @shayan-binary regarding passing testid to Button instead, as currently we can't, so opted to pass it to the icon

Reply:
let's raise this in `deriv_ui_discussion`

Reply:
ah i've raised it to shayan already, he will be doing it tmrw 🙏